packages/react-router/lib/dom/ssr/components.tsx

```diff
@@ -264,17 +268,28 @@ export function Links() {
 }
 
 /**
-  Renders `<link rel=prefetch|modulepreload>` tags for modules and data of another page to enable an instant navigation to that page. {@link LinkProps.prefetch | `<Link prefetch>`} uses this internally, but you can render it to prefetch a page for any other reason.
-
-  ```tsx
-  import { PrefetchPageLinks } from "react-router"
-
-  <PrefetchPageLinks page="/absolute/path" />
-  ```
-
-  For example, you may render one of this as the user types into a search field to prefetch search results before they click through to their selection.
-
-  @category Components
+ * Renders `<link rel=prefetch|modulepreload>` tags for modules and data of
+ * another page to enable an instant navigation to that page.
+ * [`<Link prefetch>`](../../components/Link#prefetch) uses this internally, but
+ * you can render it to prefetch a page for any other reason.
+ *
+ * For example, you may render one of this as the user types into a search field
+ * to prefetch search results before they click through to their selection.
+ *
+ * @example
+ * import { PrefetchPageLinks } from "react-router";
+ *
+ * <PrefetchPageLinks page="/absolute/path" />
+ *
+ * @public
+ * @category Components
+ * @mode framework
+ * @param props Props
+ * @param props.page The absolute path of the page to prefetch, e.g. `/absolute/path`.
+ * @param props.dataLinkProps Additional props to pass to the
```

Comment:
@brophdawg11 This is not an actual prop you can pass, it's a collection of props (it's used as a rest operator)

Reply:
yeah I was hoping the signature right above showing the rest/spread would make that apparent - but we could adjust the wording to make it more explicit?

<img width="1640" height="868" alt="Screenshot 2025-07-21 at 5 18 05 PM" src="https://github.com/user-attachments/assets/dc3f1ffe-caea-4de2-a430-5c5c860f58d3" />


I don't like [what we're doing today](https://reactrouter.com/api/components/PrefetchPageLinks) by trying to document each individual property since we're just being a worse MDN at that point and I'd rather just link out to them.